index.html

```diff
@@ -3046,26 +3054,22 @@ <h4><code>subscribeevent</code></h4>
               <code>Accept</code> header set to <code>application/json</code>
             </li>
             <li>
-              A body with a subscription payload, serialized in JSON.
+              A body with a subscription request payload, serialized in JSON.
             </li>
-
-            <p class="ednote">
-              <!-- TODO --> TODO: Define subscription payload.
+            <p>
+              The subscription payload contains the URI for the event message listener
+              in the field with the <code>callbackURI</code> key. 
```

Comment:
[Example 44](https://w3c.github.io/wot-profile/#example-44) uses `callbackURL` but this says `callbackURI`.

[Example 70](https://w3c.github.io/wot-thing-description/#example-70) in the TD spec also uses `callbackURL`.